src/app/todo-list/checkboxList.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import * as React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Checkbox from '@mui/material/Checkbox';
+
+export default function CheckboxList() {
+    const [checked, setChecked] = React.useState([-1]);
```

Comment:
一點點看法，可以參考看看

1. 形容詞加上主詞的話，可以讓這個state的作用更明確或許checkedIdList或checkedTodoIdList
2. 如果加上type可以更容易看出來這個state是代表什意義，雖然有些時候我敢進路也常常忘了加
3. 初始值方面，我自己比較習慣用null去代表是初始狀態，這樣的好處是當出現空陣列我也可以知道這是更動後的狀態
4. 不確定[-1]的作用是什麼，感覺是拿來當flag，如果是的話會建議用別的方式，因為常常會用array.length來判斷array是否為空之類的，在這種情況下就會失效

```suggestion
    const [checkedTodoIdList, setCheckedTodoIdList] = React.useState<string[] | null>(null);
```

Reply:
https://mui.com/material-ui/react-list/#CheckboxList.tsx
抄官方的 😂 我改一下

Reply:
Fixed 61d65123744972f1ad0679ba3283704c9f42d024